scripts/tests/unit/test_schema_cleaner.py

```diff
@@ -200,6 +200,7 @@ def test_field_defaults(self):
             'level': 'extended',
             'name': 'my_field',
             'type': 'unknown',
+            'index': 'true',
```

Comment:
I don't think you should add this there, it could invalidate the tests on lines 219-225.

Reply:
Thanks, corrected.